yt_dlp/extractor/wistia.py

```diff
@@ -39,6 +40,21 @@ def _extract_media(self, embed_config):
         video_id = data['hashedId']
         title = data['name']
 
+        def get_real_ext(url):
+            res = self._request_webpage(HEADRequest(url), None, note=False, errnote=False)
```

Comment:
note=errnote=False creates a complete lack of feedback. I recommend keeping a note

Also, we need `fatal=False`